task_11/build/index.html

```diff
@@ -0,0 +1,300 @@
+                    </span>
+                </label>
+                <label for="pro-tone">
+                    <input type="radio" id="pro-tone" name="website-radio-button" required>
+                    <span class="main-content__select-tone-img-wrapper">
+                    <img src="img/icons/suitcase.svg">
+                        <span>Corporate / Pro</span>
+                    </span>
+                </label>
+                <label for="media-tone">
+                    <input type="radio" id="media-tone" name="website-radio-button" required>
+                    <span class="main-content__select-tone-img-wrapper">
+                        <img src="img/icons/sound.svg">
+                        <span>Media / Formal</span>
+                    </span>
+                </label>
+            </section>
+
+            <section class="main-content__urgent-range">
+                <h2>Tell us how urgent this job is</h2>
+                <input type="range" id="urgent-range">
+                <div class="main-content__urgent-range-labels-wrapper">
+                    <label for="urgent-range">I got time</label>
+                    <label for="urgent-range" class="main-content__urgent-range-orange-input">Average</label>
+                    <label for="urgent-range">Yesterday</label>
+                </div>
+                <div class="main-content__timeline-info">
+                    <h3><img src="img/icons/info.svg">Timeline info</h3>
+                    <p>with an <span>Average</span> timeline set up, our expert translator can take a reasonable amount
+                        of time perfecing your translation</p>
+                </div>
+            </section>
+
+            <section class="main-content__choose-payment">
+                <h2>Sign up and choose a payment method</h2>
+                <div class="main-content__account-sign-in">
+                    Already have an account with us?
+                    <a href="/">Sign in.</a>
+                </div>
+                <div class="main-content__personal-payment-details">
+                    <div class="main-content__personal-payment-details-name">
+                        <label for="name">Full name</label>
+                        <input type="text" id="name" placeholder="Full name" required>
+                    </div>
+                    <div class="main-content__personal-payment-details-address">
+                        <div class="main-content__personal-payment-details-column">
+                            <label for="email">Email address</label>
+                            <input type="email" id="email" placeholder="example@test.com" required>
+                        </div>
+                        <div class="main-content__personal-payment-details-column">
+                            <label for="password">Chose a password</label>
+                            <input type="text" id="password" placeholder="*********" required minlength="8">
```

Comment:
you could use semantic tag `<input type="password">`
and each character will be replacing with a dot automatically

Reply:
Oh thanks! I missed this type.